CenterForOpenScience/osf.io
# [ENG-2814] Change draft registration listing perms

## Purpose

Changes the perms for listing draft registrations

## Changes

- Simply changes perm class and adds test

## QA Notes

Please make verification statements inspired by your code and what your code touches.
- Verify the error is no longer present


What are the areas of risk?

Any concerns/considerations/questions that development raised?

## Documentation



## Side Effects

None that I know of.

## Ticket

https://openscience.atlassian.net/browse/ENG-2814
